src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -649,6 +727,9 @@ def handle(self, *args, **options):
         if not keystone_client_exists:
             logger.error("keystoneclient module is not installed")
             raise ImportError("No module named keystoneclient")
+        if not ironic_client_exists:
```

Comment:
this should be required only in case of matching with ironic, so put this cmd argument here to `if args.match_ironic_physical_host and not ironic_client_exists` etc